src/foremast/app/create_app.py

```diff
@@ -85,13 +86,22 @@ def create_app(self, pipeline_configs=None):
         Raises:
             AssertionError: Application creation failed.
         """
+        self.pipeline_configs = pipeline_configs
         self.appinfo['accounts'] = self.get_accounts()
         self.log.debug('App info:\n%s', pformat(self.appinfo))
-
+        self.add_to_instance_links()
         jsondata = get_template(template_file='infrastructure/app_data.json.j2',
-                                appinfo=self.appinfo, pipeline_configs=pipeline_configs)
+                                appinfo=self.appinfo, pipeline_configs=self.pipeline_configs)
 
         wait_for_task(jsondata)
 
         self.log.info("Successfully created %s application", self.appname)
         return
+
+    def add_to_instance_links(self):
+        """Appends on existing instance links"""
+        if self.pipeline_configs['type'] == 'ec2':
```

Comment:
There are no cluster's view for any application other than those that are ec2. So we would not have a Links window for us :). This if statement isn't necessary because issues  shouldnt arise if we added the links even if the app isnt of ec2 type. Lets see what @ndcampbell thinks